viewdir/index.njk

```diff
@@ -1,6 +1,132 @@
+{% extends "layout-template.njk" %}
+
+{%block pageTitle%}
+    Login
+{% endblock %}
+
 {% block content%}
-<h1>{{ email }}</h1>
-<form action='/logout' method="GET">
-    <input type="submit" value="Logout" />
-</form>
-{% endblock%}
\ No newline at end of file
+    <head>
+        <title>Kainos - Home</title>
+    </head>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <body>
+        <div class="container" style="margin-top: 2%">
+            <div class="row">
+                <div class="col s4">
+                    <a href="http://localhost:6555/JobsTable">
+                        <div class="card-panel hoverable" style="background-color: #61A83F ; margin-top: 10%">
```

Comment:
Normally we don't want any inline `style` attributes; styles should normally be defined in .css files